add new articles for using binaries (#12216)
openvinotoolkit/openvino: docs/install_guides/uninstalling-openvino.md

## Uninstall Using the Original Installation Package

> If you have installed OpenVINO Runtime from archive files, use the following commands to uninstall it:

Reviewer: ```suggestion
If you have installed OpenVINO Runtime from archive files, you can uninstall it by deleting the zip file and extracted folder.
```

@sphinxdirective
.. tab:: Windows

  You can choose either of the following methods:

>   * Use the Windows GUI to remove the extracted files.

Reviewer: ```suggestion
  * Use Windows Explorer to remove the files.
```

>   * Open the Command window and run:

Reviewer: ```suggestion
  * Open a Command Prompt and run:
```

     .. code-block:: sh
  

>        rmdir <extracted_folder>

Reviewer: ```suggestion
       rmdir /s <extracted_folder>
```
Without `/s` you will get the error that the directory is not empty
